src/parquet/file_writer.h

```diff
@@ -50,8 +54,16 @@ class PARQUET_EXPORT RowGroupWriter {
     virtual int64_t num_rows() const = 0;
 
     virtual ColumnWriter* NextColumn() = 0;
+    // to be used only when row_group_by_size = true
```

Comment:
Clarify in a comment for `NextColumn()` that `ColumnWriter*` objects become invalid when `NextColumn` is called unless writing fixed size row groups